src/test/scala/no/ndla/searchapi/service/search/MultiSearchServiceTest.scala

```diff
@@ -372,6 +372,7 @@ class MultiSearchServiceTest extends IntegrationSuite with TestEnvironment {
     val Success(search) =
       multiSearchService.matchingQuery(searchSettings.copy(subjects = List("urn:subject:2"), language = "all"))
     search.totalCount should be(7)
+    search.results.head.contexts.length should be(2)
```

Comment:
Skal returnere kontekst for subject:1 og subject:2, som spesifisert i linje 1024 og 1025. Kontekst for subject:3 i linje 1040 filtreres ut.

Reply:
Kan se for meg at bare en test på antall kan føre til at vi ikke merker en regresjon her, er det en ide og teste det litt mer eksplisitt med å feks gjøre noe sånt?

```suggestion
    search.results.head.contexts.map(_.id) should be(Seq(...))
```

Reply:
Oppdaterte testen med sjekk på subjectId.